src/ontology/uberon-edit.obo

```diff
@@ -222238,6 +222238,16 @@ synonym: "fifth molar tooth" EXACT []
 is_a: UBERON:0003655 ! molar tooth
 relationship: dc-contributor https://orcid.org/0000-0003-2699-3066
 
+[Term]
+id: UBERON:7500128
+name: cranial temporal crest
+def: "A dorsally concave, laterally-directed crest that terminates at the caudal-dorsal process of a zygomatic bone. It is bounded by the superior and inferior temporal lines." [DOI:10.5455/ijlr.20170908094210]
+synonym: "temporal crest" BROAD []
```

Comment:
> Does a broad synonym to "temporal bone" make more sense?

Assuming you mean "temporal crest", perhaps **related synonym** would be appropriate. 
This is because you are defining the term based on a paper that calls it the "temporal crest", so that term means exactly what your are defining, but that term may also refer to other "temporal crests", like as you noted above, on the mandible. Considering the potential ambiguity (the term may refer to exactly what you are defining or to a completely separate structure), related may be the "safest" choice.